passageidentity/passage-python
# chore: pin dependencies to latest

## What's New?



- Pins deps to latest
- Removes unused deps
- Removes deps that are already pulled in by the libraries using them

To test:
1. Open project in dev container
2. `pytest`
3. All tests should pass
4. Try removing a dependency from the new list
5. Rebuild dev container
6. `pytest`
7. Tests should fail with an error that the package cannot be found

## Screenshots (if appropriate):



## Type of change


## Checklist:


## Additional context
